inlanecruising_plugin/include/inlanecruising_plugin/inlanecruising_plugin_node.h

```diff
@@ -69,12 +73,40 @@ class InLaneCruisingPluginNode
     
     InLaneCruisingPlugin worker(wm_, config, [&discovery_pub](auto msg) { discovery_pub.publish(msg); });
 
+
+    
+
+    ros::ServiceClient yield_client = nh.serviceClient<cav_srvs::PlanTrajectory>("plugins/YieldPlugin/plan_trajectory");
```

Comment:
Not a big deal now, but let's make a note to update this later to use the Plugin Manager capabilities query, in case someone else wants to add an alternate yield implementation